src/libraries/Microsoft.PowerFx.Core/Parser/TexlParser.cs

```diff
@@ -87,13 +87,14 @@ public static ParseFormulasResult ParseFormulasScript(string script, ILanguageSe
 
         private ParseFormulasResult ParseFormulas(string script)
         {
-            var namedFormulas = new List<KeyValuePair<IdentToken, TexlNode>>();
+            var namedFormulas = new List<NamedFormula>();
             ParseTrivia();
-
+            var offset = _curs.CurrentCharIndex;
```

Comment:
>var offset = _curs.CurrentCharIndex; [](http://example.com/codeflow?start=12&length=36)

can we delete this?
offset is only used in the nested scope.

Reply:
yeah, it can  be deleted.